pkg/virt-handler/vm_test.go

```diff
@@ -2699,21 +2705,39 @@ var _ = Describe("VirtualMachineInstance", func() {
 			conditionManager := virtcontroller.NewVirtualMachineInstanceConditionManager()
 			controller.updateLiveMigrationConditions(vmi, conditionManager)
 
-			testutils.ExpectEvent(recorder, fmt.Sprintf("cannot migrate VMI which does not use masquerade to connect to the pod network or bridge with %s VM annotation", v1.AllowPodBridgeNetworkLiveMigrationAnnotation))
+			testutils.ExpectEvent(recorder, fmt.Sprintf("cannot migrate VMI which does not use masquerade, bridge with %s VM annotation or a migratable plugin to connect to the pod network", v1.AllowPodBridgeNetworkLiveMigrationAnnotation))
 		})
-		Context("with AllowLiveMigrationBridgePodNetwork annotation", func() {
-			It("should allow to live-migrate if the VMI use bridge to connect to the pod network", func() {
-				vmi := api2.NewMinimalVMI("testvmi")
-
-				vmi.Annotations = map[string]string{v1.AllowPodBridgeNetworkLiveMigrationAnnotation: ""}
-
+		Context("should allow migration", func() {
+			var vmi *v1.VirtualMachineInstance
+			var conditionManager *virtcontroller.VirtualMachineInstanceConditionManager
+			BeforeEach(func() {
+				vmi = api2.NewMinimalVMI("testvmi")
```

Comment:
Please start using `libvmi` here or as a follow up.
This change extends the existing usage and we would like to stop its usage.

It will also change the code, e.g. `vmi` will not be defined here but in the tests themselves.